src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -564,14 +565,8 @@ private CallResult ExecutePrecompile(EvmState state, IReleaseSpec spec)
             _parityTouchBugAccount.ShouldDelete = true;
         }
 
-        //if(!UpdateGas(dataGasCost, ref gasAvailable)) return CallResult.Exception;
-        if (!UpdateGas(baseGasCost, ref gasAvailable))
-        {
-            Metrics.EvmExceptions++;
-            throw new OutOfGasException();
-        }
-
-        if (!UpdateGas(dataGasCost, ref gasAvailable))
+        if (!UpdateGas(baseGasCost, ref gasAvailable) ||
+            !UpdateGas(dataGasCost, ref gasAvailable))
```

Comment:
Minor `!UpdateGas(baseGasCost*2, ref gasAvailable)`  or `!UpdateGas(baseGasCost+baseGasCost, ref gasAvailable)` or `!UpdateGas(baseGasCost<<1, ref gasAvailable)`? Probably same IL.

Would it change anything in trace?

Reply:
Nope not traced

Reply:
Will do `checked(baseGasCost + dataGasCost)`